scripts/ts-bin/multiplex_transform.ts

```diff
@@ -96,15 +97,16 @@ export class Multiplex extends stream.Transform {
     writable.on('unpipe', () => { console.debug(`${streamName}: unpipe`); });
   }
 
-  pushToQueue(streamName:string, chunk:Buffer|string|{}, encoding:string, cb:() => void) {
+  pushToQueue(streamName: string, chunk: Buffer|string|{}, encoding: string,
+      cb: () => void) {
     this.push(streamName);
     let isNotFull = this.outputStreams[streamName].write(chunk, encoding, cb);
     if(!isNotFull) {
       this.fullStreams[streamName] = null;
     }
   }
 
-  _transform(chunk:string, encoding:string, cb:stream.TransformCallback) : void {
+  _transform(chunk: string, encoding: string, cb: Function) : void {
```

Comment:
Is it possible to more strongly type the cb: Function argument?

Reply:
no, it's typed by the class, and it specifies Function.